### ISSUE TITLE
accessible view should not show unless focused element has registered provider

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#186521]
Running the open accessible view command doesn't do anything for me. It doesn't work regardless of whether screen reader mode is on

Version: 1.80.0-insider (user setup)
Commit: 4649b5333370139d161e7b5d4f78ca13741efa35
Date: 2023-06-28T08:01:04.380Z
Electron: 22.3.14
ElectronBuildId: 21893604
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621

**Comment by meganrogge** [comment]
It won't do anything unless you have a hover or a chat response focused

**Comment by meganrogge** [comment]
what is focused when you tried?

**Comment by Tyriar** [comment]
That was with focus in terminal, editor and sidebar. I tried just now with the terminal focused (chat visible) and it showed some very unexpected text:

![image](https://github.com/microsoft/vscode/assets/2193314/457157dc-c808-4deb-9400-2db08b95d36d)

**Comment by meganrogge** [comment]
With focus in a hover or chat response, it should show. Otherwise, it shouldn't.

**Comment by Tyriar** [comment]
@meganrogge this one doesn't seem too critical so could move to July?

**Comment by meganrogge** [comment]
Well, it could be disorienting for SR users if it's showing content unexpectedly. But yeah probably risky at this point

**Comment by meganrogge** [comment]
actual fix will be in the linked PR, but a related issue will need to be resolved before this works fully #187729

**Comment by Tyriar** [comment]
Works generally now, I found this edge case though https://github.com/microsoft/vscode/issues/188954
